### PR TITLE
fix(assigneeSelector): fix styles and errors

### PR DESCRIPTION
**Pull request by MaxBittker**
fixes JAVASCRIPT-384
fixes JAVASCRIPT-361

### CODE REVIEW

File: src/sentry/static/sentry/app/components/assigneeSelector.jsx [hunk 1]
```diff
@@ -79,7 +79,7 @@ const AssigneeSelector = createReactClass({
     if (nextProps.id != this.props.id || loading != this.state.loading) {
       let group = GroupStore.get(this.props.id);
       this.setState({
-        assignedTo: group.assignedTo,
+        assignedTo: group && group.assignedTo,
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/assigneeSelector.jsx]: Do we know why `group` can be undefined here?

File: src/sentry/static/sentry/app/components/assigneeSelector.jsx [hunk 2]
```diff
@@ -212,8 +212,19 @@ const AssigneeSelector = createReactClass({
   render() {
     let {loading, assignedTo, filter, memberList} = this.state;
     let memberListLoading = this.state.memberList === null;
+    let group = GroupStore.get(this.props.id);
+
+    if (loading || !group) {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/assigneeSelector.jsx]: Why are there 2 loading states?

File: src/sentry/static/sentry/app/components/assigneeSelector.jsx (hunk 1 above)
Comment (MaxBittker) [reply src/sentry/static/sentry/app/components/assigneeSelector.jsx]: I think it's a race condition on load? to be honest I don't so maybe this fix is too hasty

**Review by billyvg — APPROVED** [review]
Can you post SS of css changes?

Otherwise refactor seems to pass tests so lgtm

**Comment by MaxBittker** [comment]
they're on percy if you're curious, but they're too small to see in a screenshot without onionskin before/after

**Pull request merged**